sdk/rust-examples/prime-numbers-generator/generate-set/src/main.rs

```diff
@@ -0,0 +1,45 @@
+//! Prime Numbers Generator: Generate the number set
+//!
+//! ## Context
+//!
+//! Generate all numbers from 2 to the specified upper limit
+//! 
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSING.markdown` in the Veracruz root directory for licensing and
+//! copyright information.
+
+use std::fs;
+use anyhow;
+
+const OUTPUT_FILENAME: &'static str = "/output/number-set.txt";
+
+fn main() -> anyhow::Result<()> {
+    let mut set: Vec<u32> = Vec::new();
+
+    let args: Vec<String> = std::env::args().collect();
+    let upper_limit = args.get(0)
+        .unwrap()
```

Comment:
What happens if there is no argument given?  What does `.unwrap()` do in that case?

Reply:
I changes it to 
```
    let upper_limit = args.get(0).unwrap_or(&String::from("100")).to_owned().parse::<u32>()?;
```
since the first form requires `Deafult trait bound` to be satisfied

Reply:
Great!